src/qibo_cloud_backends/__init__.py

```diff
@@ -48,20 +51,23 @@ def list_available(self, tokens: dict = None) -> dict:
 
         Args:
             tokens (dict): Mapping between the services and their tokens, e.g.
-                           {"qibo-client": "xxxxx", "qiskit-client": "xxxxx"}.
-                           By default reads the variables ("QIBO_CLIENT_TOKEN", "IBMQ_TOKEN").
+                           {"qibo-client": "xxxxx", "qiskit-client": "xxxxx", "braket-client": "xxxxx"}.
+                           By default reads the variables ("QIBO_CLIENT_TOKEN", "IBMQ_TOKEN", None).
         Returns:
             dict: the qibo-cloud available backends.
         """
         if tokens is None:
             tokens = {}
         available_backends = {}
         for client, token in zip(CLIENTS, TOKENS):
+            kwargs = {}
+            if client != "braket-client":
+                token = tokens.get(client, os.environ.get(token))
+                kwargs.update({"token": token})
```

Comment:
It was just `token` before. But @BrunoLiegiBastonLiegi suggested to use `kwargs` [here](https://github.com/qiboteam/qibo-cloud-backends/pull/30#discussion_r1777098948). I presume it might be for additional arguments in future?

Reply:
The `aws-client` backend doesn't accept any `token` input argument, the `kwargs` is there just so that we can build the correct backend-dependent `kwargs` before and avoid having an:
``` python
if something:
    try: ...
    except: ...
else:
    try: ...
    except: ...
```

Reply:
I see...

But, since these are backends we're making ourselves, couldn't we standardize the arguments from the backends side?

Like, add the `token` argument handling in the `BraketClientBackend`, and raise if you receive a value different from `None`.

The logic is that this is the general scope, but we already have a specific one. So, backends' specific features could be handled in the respective backends' modules.

Reply:
Yeah in my view, it would be optimal to have the backends as uniform as possible, thus including a `token=None` argument to the `aws-client` and even replacing the current `device` argument with the `platform` one that is used by the other two. But I did not want to force this too much as `device` is also the original name used by AWS and I did not want to risk to cause confusion.